src/cockpit/channels/metrics.py

```diff
@@ -120,35 +120,42 @@ def sample(self):
             sampler.sample(samples)
         return samples
 
-    def calculate_sample_rate(self, value: Any, old_value: Optional[Any]):
+    def calculate_sample_rate(self, value: float, old_value: Optional[float]) -> Union[float, bool]:
```

Comment:
That's actually `Literal[False]`, isn't it?

Reply:
Yes, but Python 3.6